tests/compile-fail/chdir_invalid_path.rs

```diff
@@ -0,0 +1,12 @@
+// ignore-windows: TODO the windows hook is not done yet
```

Comment:
weeelll... this is just a C function in this test, so it'll "work" on windows, too

Reply:
Oh well that's true, do you want me to unignore windows?

Reply:
yea, no harm in doing so